spec/numberformat.html

```diff
@@ -641,7 +641,7 @@ <h1>PartitionNotationSubPattern ( _numberFormat_, _x_, _n_, _exponent_ )</h1>
             1. Else if _p_ is equal to *"number"*, then
               1. If the _numberFormat_.[[NumberingSystem]] matches one of the values in the *"Numbering System"* column of <emu-xref href="#table-numbering-system-digits"></emu-xref> below, then
                 1. Let _digits_ be a List whose 10 String valued elements are the UTF-16 string representations of the 10 _digits_ specified in the *"Digits"* column of the matching row in <emu-xref href="#table-numbering-system-digits"></emu-xref>.
-                1. Replace each _digit_ in _n_ with the value of _digits_[_digit_].
+                1. [declared="digit"] Replace each _digit_ in _n_ with the value of _digits_[_digit_].
```

Comment:
This is no worse than the current text, but it is highlighting a problem that I think we can avoid by following the precedents of [ASCII operations](https://tc39.es/ecma402/#sec-case-sensitivity-and-case-mapping) and [GetSubstitution](https://tc39.es/ecma262/multipage/text-processing.html#sec-getsubstitution).
```suggestion
                1. [declared="digit"] Set _n_ to the String value derived from _n_ by replacing each occurrence of an ASCII digit code unit (0x0030 through 0x0039, inclusive) _digit_ with the corresponding sequence of code units from _digits_[_digit_ - 0x0030].
```
or even more explicitly:
```suggestion
                1. Let _transliterated_ be the empty String.
                1. Let _len_ be the length of _n_.
                1. Let _position_ be 0.
                1. Repeat, while _position_ &lt; _len_,
                  1. Let _c_ be the code unit at index _position_ within _n_.
                  1. If 0x0030 ≤ _c_ ≤ 0x0039, then
                    1. NOTE: _c_ is an ASCII digit.
                    1. Let _i_ be _c_ - 0x0030.
                    1. Set _c_ to _digits_[_i_].
                  1. Set _transliterated_ to the string-concatenation of _transliterated_ and _c_.
                  1. Set _position_ to _position_ + 1.
                1. Set _n_ to _transliterated_.
```